### PR TITLE
3.2.x: Downloads: mender-client: Update URLs for direct downloads

### PR DESCRIPTION
**Pull request by lluiscampos**
Following packages names updates from MEN-5410.

(cherry picked from commit 8f0345f7d1992b24bf09e2bc393a011dd92c099c)

Setting client version to 3.2.1 while resolving merge conflict.

### CODE REVIEW

File: 09.Downloads/docs.md [hunk 1]
```diff
@@ -258,21 +258,21 @@ airtight systems with limited access to the Internet, or when running
 Mender in [standalone
 mode](../02.Overview/01.Introduction/docs.md#client-modes-of-operation).
 
-<!--AUTOVERSION: "mender-client_%-1"/mender -->
+<!--AUTOVERSION: "mender-client_%_a"/mender "mender-client_%-1"/mender -->
 | Architecture   | Devices                                                                                        | Download link                                                       | Download link                                                                      |
```
Comment (michaelatmender) [inline 09.Downloads/docs.md:262]: Is it intentional to have 2 "download link" columns on this table?

File: 09.Downloads/docs.md [hunk 2]
```diff
@@ -258,21 +258,21 @@ airtight systems with limited access to the Internet, or when running
...
-[mender-client_x.x.x-1_armhf.deb]: https://downloads.mender.io/3.2.1/dist-packages/debian/armhf/mender-client_3.2.1-1_armhf.deb
-[mender-client_x.x.x-1_arm64.deb]: https://downloads.mender.io/3.2.1/dist-packages/debian/arm64/mender-client_3.2.1-1_arm64.deb
-[mender-client_x.x.x-1_amd64.deb]: https://downloads.mender.io/3.2.1/dist-packages/debian/amd64/mender-client_3.2.1-1_amd64.deb
-<!--AUTOVERSION: "downloads.mender.io/%/"/ignore "mender-client_%-1_"/ignore -->
-[mender-client_master-1_armhf.deb]: https://downloads.mender.io/master/dist-packages/debian/armhf/mender-client_master-1_armhf.deb
-[mender-client_master-1_arm64.deb]: https://downloads.mender.io/master/dist-packages/debian/arm64/mender-client_master-1_arm64.deb
-[mender-client_master-1_amd64.deb]: https://downloads.mender.io/master/dist-packages/debian/amd64/mender-client_master-1_amd64.deb
+| armhf (ARM-v6) | ARM 32bit distributions, for example Raspberry Pi OS for Raspberry Pi or Debian for BeagleBone | [mender-client_3.2.1-1+debian+buster_armhf.deb][mender-client_x.x.x_armhf.deb] | [mender-client_master-1+debian+buster_armhf.deb][mender-client_master_armhf.deb] (Pre-release) |
```
Comment (michaelatmender) [inline 09.Downloads/docs.md:264]: The formatting of the links still doesn't seem right - it should be `[link text](url)`

File: 09.Downloads/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 09.Downloads/docs.md:262]: Yes. The stable version for this branch (in our case, `3.2.1`) and the master (pre-release). See [here](https://docs.mender.io/development/downloads#install-from-package).

File: 09.Downloads/docs.md (hunk 2 above)
Comment (lluiscampos) [reply 09.Downloads/docs.md:264]: They are correct, it is the syntax for expanding the actual links below. See for example `mender-artifact` of `mender-cli` tables in this same page. And then see rendered tables [here](https://docs.mender.io/3.2/downloads#mender-artifact) and [here](https://docs.mender.io/3.2/downloads#mender-cli).

File: 09.Downloads/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 09.Downloads/docs.md:262]: Same as for `mender-artifact` and `mender-cli`. And then see rendered tables [here](https://docs.mender.io/3.2/downloads#mender-artifact) and [here](https://docs.mender.io/3.2/downloads#mender-cli).

File: 09.Downloads/docs.md (hunk 2 above)
Comment (michaelatmender) [reply 09.Downloads/docs.md:264]: Thanks for the explanation! Probably I just don't understand how the autoversioning works, I was just thinking about markdown.

**Review by michaelatmender — APPROVED** [review]

File: 09.Downloads/docs.md (hunk 2 above)
Comment (lluiscampos) [reply 09.Downloads/docs.md:264]: It is not autoversion magic though, [it is markdown syntax](https://www.markdownguide.org/basic-syntax/#reference-style-links).

**Pull request merged**